src/stores/modules/TaskModule.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+
+export default {
+    namespaced: true,
+    state() {
+        return {
+            tasks: [],
+            requestProcessing: false,
+            baseUrl: 'http://localhost:3000/task/',
+        };
+    },
+    mutations: {
+        setRequestProcessing(state) {
+            state.requestProcessing = true;
+        },
+        resetRequestProcessing(state) {
+            state.requestProcessing = false;
+        },
+        setTasks(state, tasks) {
+            state.tasks = [...tasks];
+        },
+        addTask(state, task) {
+            state.tasks.push(task);
+        },
+        deleteTask(state, id) {
+            state.tasks = state.tasks.filter((task) => task.id !== id);
+        },
+    },
+    actions: {
+        async retriveTasks({commit, state}) {
+            return await axios.get(state.baseUrl)
+                .then((result) => {
```

Comment:
Czy byl juz poruszany temat error handlingu?

Reply:
Jeszcze nie uczyłem się error handlingu. Warto teraz to tego siąść czy narazie zostawić?